client/src/components/Admin/Features.js

```diff
@@ -48,21 +51,39 @@ const Features = () => {
     refetch: featureToLoginRefetch,
   } = useFeatureToLogin();
 
+  const { data: featuresData, loading: featuresLoading } = useFeatures();
+
   useEffect(() => {
-    const newRows = featureToLoginData.map((feature) => ({
-      featureId: feature.feature_id,
-      name: feature.feature_name,
-      featureToLoginId: feature.ftl_id,
-      history: feature.users.map((user) => ({
-        loginId: user.login_id,
-        firstName: user.first_name,
-        lastName: user.last_name,
-        email: user.email,
-        featureToLoginId: user.featureToLoginId || feature.ftl_id,
-      })),
-    }));
+    if (featureToLoginLoading || featuresLoading) {
```

Comment:
we could just check for `featuresData` before we use it. Returning something from a useEffect isn't necessary unless we need to do some clean up 🤔